docs/advanced-use.rst

```diff
@@ -0,0 +1,157 @@
+.. _advanced-use:
+
+Data pre-processing and troubleshooting
+=======================================
+
+Pre-processing your data
+------------------------
+
+Manipulating the text and/or audio data that you are trying to align can
+sometimes produce longer, more accurate ReadAlongs, that throw less
+errors when aligning. While some of the most successful techniques we
+have tried are outlined here, you may also need to customize your
+pre-processing to suit your specific data.
+
+Audio pre-processing
+~~~~~~~~~~~~~~~~~~~~
+
+Adding silences
```

Comment:
I didn't know we did this... then we have to undo this though in the eventual readalong right? This sounds a bit unnecessary if we have anchors.

Reply:
Fin used it quite a bit, but it's true with the anchors feature this might be obsolete. I think where she did this, she kept the inserted silences in the results.